java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ConnectionTest.java

```diff
@@ -254,9 +237,10 @@ public void testUnencryptedConnectionShouldOpenSuccessfullyWithoutAuthentication
       throws Exception {
     final Properties properties = new Properties();
     properties.put(ArrowFlightConnectionProperty.HOST.camelName(), "localhost");
-    properties.put(ArrowFlightConnectionProperty.PORT.camelName(), server.getPort());
+    properties.put(ArrowFlightConnectionProperty.PORT.camelName(),
+        dataSource.getConfig().getPort());
     try (Connection connection = DriverManager
-        .getConnection(serverUrl, properties)) {
+        .getConnection("jdbc:arrow-flight://localhost:32010", properties)) {
```

Comment:
Is this test working? What I am reading by the test name is that you want to create a connection without using authentication. However, you are starting a server that needs authentication, so should this fail?

Reply:
This test is correct. This test method is try to make connect unscripted and not necessary authentication.

Reply:
I think is this...

Reply:
This test is correct. This test method is try to make connect unscripted and not necessary authentication.

I think is this...